docker-compose.yml

```
@@ -11,12 +11,9 @@ services:
     db:
         image: mysql:8
         volumes:
-            - db:/var/lib/mysql:cached
+            - ./.db:/var/lib/mysql:cached
```

```suggestion
            - ./docker/.db:/var/lib/mysql:cached
```

wollen wir dann nicht alles docker relevante in den `/docker` ordner schieben

---

Du meinst `.docker` (mit Punkt), oder? Zumindest steht der Ordner so in der gitignore.
Vielleicht dann `.docker/db`, also `db` dann ohne Punkt?

---

Sorry, ich hatte den Vorschlag am Telefon nicht erkannt und nur auf den Kommentar geachtet. Hab’s gerade so umgesetzt in der Form `.docker/db`.

---

Arrg, genau meinte ich mit Punkt :see_no_evil: